journal.py

```diff
@@ -203,6 +221,22 @@ def details(request):
     return {'entries': entry}
 
 
+@view_config(route_name='tweet', renderer='json')
+def tweet_all_about_it(request):
+
+    consumer_key = 'wg0IdOvNPO6SwKNaTz7MWI9J8'
+    consumer_secret = 'OE71lxs2PqOC3hBlIwK85l6npHGsuCmwX0WScAriA4JWFqiVAe'
+    access_token = '3008504700-7ZJGjPtwOZDwEsLHBh4oyoJQdswqmLSyj1QLJI3'
+    access_token_secret = 'UNCrF1nZkurJPpT1ZSACUrGJvwFPxMH4On79Clr611OOa'
+
+    auth = tweepy.OAuthHandler(consumer_key, consumer_secret)
+    auth.set_access_token(access_token, access_token_secret)
+
+    api = tweepy.API(auth)
+
+    api.update_status(status=request.params['title'])
```

Comment:
Instead of just tweeting your title, you might also want to add a link to the detail view for this post.  That'd let people read about what you're up to.